docs/realtime/index.md

```diff
@@ -1,7 +1,7 @@
 # GTFS Realtime Overview
 
-<div class="landing-page">
-    <a class="button" href="reference">Reference</a><a class="button" href="best-practices">Best Practices</a><a class="button" href="examples">Examples</a><a class="button" href="changes">Changes</a>
+<div class=landing-page>
```

Comment:
Is this valid HTML?

Reply:
The static website generator used for this project (MkDocs) renders it: https://gtfs.org/realtime/

The css is coming from https://github.com/MobilityData/gtfs.org/blob/main/docs/assets/stylesheets/extra.css